schism/keybinds.c

```diff
@@ -0,0 +1,499 @@
+
+#include "it.h"
+#include "page.h"
+#include "dmoz.h"
+#include "keybinds_codes.c"
+#include "keybinds_init.c"
```

Comment:
Reason why I didn't do that is because they are not supposed to be standalone modules, they are just a way to split `keybinds.c` into different files, cause it's too long otherwise

Reply:
`audio_playback.c` has more lines of code itself than these three files combined. separating them into different sections within the same file, which is what a lot of the other files do as well, would work fine here.